src/main/scala/org/hammerlab/guacamole/readsets/ReadSet.scala

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hammerlab.guacamole.readsets
+
+import org.apache.spark.SparkContext
+import org.bdgenomics.adam.models.SequenceDictionary
+import org.hammerlab.guacamole._
+import org.hammerlab.guacamole.reads.Read
+
+/**
+  * A ReadSet contains an RDD of reads along with some metadata about them.
+  *
+  * @param reads RDD of reads
+  */
+case class ReadSet(reads: ReadsRDD,
```

Comment:
I didn't add back the "filters" attribute because it's less clear whether that should be an attribute of individual `ReadsRDD`s, or the collective `ReadSets`; today all reads that are loaded share filters, but it doesn't seem worth exposing that as an attribute of a `ReadSets`, especially because the filters are unchanged from those passed in by the user to a `ReadSets`; everything else on a `ReadSets` is loaded from the underlying data.